src/sentry/web/frontend/debug/mail.py

```diff
@@ -340,16 +342,15 @@ def digest(request):
         offset = timedelta(seconds=0)
         for i in range(random.randint(1, 10)):
             offset += timedelta(seconds=random.random() * 120)
-            event = Event(
-                id=next(event_sequence),
-                event_id=uuid.uuid4().hex,
-                project=project,
-                group=group,
-                message=group.message,
-                data=load_data("python"),
-                datetime=to_datetime(
```

Comment:
Reason for removing this ?

Reply:
Replaced it now since we need fixed timestamps